nginx/public/node/frontend/public/js/app/search/chant-search/SearchInputView.js

```diff
@@ -66,6 +66,12 @@ export default Marionette.ItemView.extend({
             // escaped hyphens in the query string.
             searchInput = searchInput.replaceAll("-","\\-");
         }
+        // Handle quotations in text field searches. Solr errors if quotation marks
+        // are not closed. If the search string contains an odd number of quotation
+        // marks, add a quotation mark to the end of the string.
+        if (["all","feast","genre","office"].includes(searchField)){
+            (searchInput.split('"').length - 1) % 2 === 1 ? searchInput += '"' : null;
```

Comment:
Does this handle single quotation marks too, and should we be concerned about quotations within quotations?

Reply:
It just handles double marks...

The honest reason is that solr does not complain with single marks (or many other punctuation marks that you wouldn't expect in any of our text -- eg. `#` `@` `%` `&` etc). As far as I can tell, solr just strips out the single marks. 

Solr does respond to other reserved characters when entered in the search box (eg. `-` and `(`) and I do think we need a better way of making this search syntax available to the user.

In any case, single marks don't cause the error so I didn't fix them here, but it's a good point that ideally we need to figure out what's going on in these other cases.